<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proj2D_NG</class>
 <widget class="QMainWindow" name="Proj2D_NG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proj2D_NG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabGroup">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabConstr">
       <attribute name="title">
        <string>1.Constraints</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="2">
         <widget class="QGroupBox" name="groupBoxConstraints">
          <property name="title">
           <string>Constraints</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPlainTextEdit" name="plainTextContraints">
             <property name="documentTitle">
              <string/>
             </property>
             <property name="plainText">
              <string>function f(x)
  return ((2*x,-x);(-x,3*x));
end</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="parseButton">
             <property name="text">
              <string>Parse</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>2.Variables</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupVar">
          <property name="title">
           <string>Variables</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <layout class="QGridLayout" name="varGridLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item row="1" column="3">
            <widget class="QDoubleSpinBox" name="upperBoundSpinBox"/>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="intervalTypeComboBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="varComboBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="addVarButton">
             <property name="text">
              <string>Add Variable info</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QDoubleSpinBox" name="lowerBoundSpinBox"/>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="ubLabel">
             <property name="text">
              <string>Upper Bound</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="lbLabel">
             <property name="text">
              <string>Lower Bound</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="unionCompolabel">
             <property name="text">
              <string>Union/Compo</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="varNameLabel">
             <property name="text">
              <string>Variables</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="evalButton">
          <property name="text">
           <string>Evaluate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabProjVar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>3.Projected Variables</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupProjVarHor">
          <property name="title">
           <string>Horizontal Axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupProjVarVer">
          <property name="title">
           <string>Vertical Axis</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDraw">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>4.Projection</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
